traitlets/config/configurable.py

```diff
@@ -402,6 +424,10 @@ def class_config_rst_doc(cls):
                 termline += ' : ' + ttype
             lines.append(termline)
 
+            env_var = trait.metadata.get('envvar')
+            if env_var:
+                lines.append(indent(indent('Env-var: ``%s``' % env_var, 4)))
```

Comment:
Fixed. 
Were missng  also `'#  '` prefix in write-config comment-line.
Now had added new parameterized pytest TC checking new comment-line in all 3 cases.